net/packet/tsmp.go

```diff
@@ -39,6 +39,16 @@ type TailscaleRejectedHeader struct {
 	Dst    netaddr.IPPort        // rejected flow's dst
 	Proto  IPProto               // proto that was rejected (TCP or UDP)
 	Reason TailscaleRejectReason // why the connection was rejected
+
+	// MaybeBroken is whether the rejection is non-terminal (the
+	// client should not fail immediately). This is sent by a
+	// target when it's not sure whether it's totally broken, but
+	// it might be.  For example, the target tailscaled might
+	// think its host firewall or IP forwarding aren't configured
+	// properly, but they might be. When true, the message is
```

Comment:
"but they might be" has an unclear antecedent. Maybe ", but tailscaled might be wrong"?

Reply:
done, and clarified a bit more